docs/index.md

```diff
@@ -1,54 +1,17 @@
-### What Are We Fighting For?
-As 5e players, we find ourselves making choices that affect combat. Either a roleplay choice will force or avoid combat.
-Perhaps a build choice increases damage at the cost of survivability.
-Maybe you need to need to decide between casting a damage spell, a buffing spell, a control spell, or a healing spell.
-An enemy is about to attack you for sure, should you fight defensively, or attack it on your turn?
+### What should a player do in combat?
+In 5e D&D, players' choices are all related to combat in some way. A roleplay choice will force or avoid combat. Choosing class features can improve damage, survivability, allow a character to take more actions or avoid combat, and usually each one comes at the cost of another. In combat, you need to decide between casting a spell that deals damage, heals or changes your allies' or enemies' conditions. An enemy is about to attack you for sure, should you dodge, disengage or attack? If there's an objective function, you can test each option and know which choice is right.
```

Comment:
You could say "we", "you" or any other colloquialism. I wouldn't use "we find ourselves." I reserve that for when it's true in a geographic sense, ie. "After being lost we found ourselves in a clearing."